src/support/cleanse.h

```diff
@@ -8,6 +8,8 @@
 
 #include <stdlib.h>
 
+// Securely erase a span of memory in a way guaranteed not to be
```

Comment:
Agreed, I reduced the verbiage to make it clear that it is a best-effort attempt.  

Also, it is my understanding that LTO only affects the object files you are linking, so dynamically loaded .so's are still safe from compiler analysis.  But I could be wrong, I've never actually come across a project of any significant size using LTO.

Comment updated.